Makefile

```diff
@@ -39,10 +39,8 @@ check-all: check-hail check-services
 
 .PHONY: check-hail-fast
 check-hail-fast:
-	ruff check hail/python/hail
-	ruff check hail/python/hailtop
+	ruff check hail
```

Comment:
I think `ruff check` exclusively refers to the [ruff linter](https://docs.astral.sh/ruff/linter/#ruff-check). I think to check formatting you need `ruff format --check`.

Reply:
ohhhhhh thank you! i will take another look with this in mind